csearch.py

```diff
@@ -142,6 +144,16 @@ def on_finished(self, output, err=None):
         sublime.set_timeout(
             functools.partial(self._finish, output, matches, err=err))
 
+def fix_windows_output(output):
+    result = []
+    # replace ntpaths to posix
+    r = re.compile(r"^([^:]*):([^:]*):([^:]*):(.*)$")
+    for line in output.splitlines():
+        m = r.match(line)
+        if m:
+            line = ('/%s%s:%s:%s' % (m.group(1), m.group(2).replace('\\','/'), m.group(3), m.group(4)))
```

Comment:
Instead of % formatting, how about using '/{}{}:{}:{}'.format(...). That would be consistent for how string formatting is done in this project.